app/core/tests/conftest.py

```diff
@@ -52,7 +52,17 @@ def project():
 
 @pytest.fixture
 def event(project):
-    return Event.objects.create(name="Test Event", project=project)
+    return Event.objects.create(
+        name="All",
+        project=project,
+        must_attend=(
+            '[{"practice_area":"Professional Development","permisstion_type":"adminProject"},'
+            '{"practice_area":"Development","permisstion_type":"practiceLeadProject"},'
+            '{"practice_area":"Design","permisstion_type":"practiceLeadJrProject"}]'
+        ),
+        should_attend='[{"practice_area":"Development","permission_type":"memberProject"}]',
+        could_attend='[{"practice_area":"Design","permission_type":"memberGeneral"}]',
```

Comment:
the `permission_type` is misspelled. Same for several other places.

Reply:
You should pass in JSON objects rather than the string representation of it, like this
```
        must_attend=[
            {"practice_area": "Professional Development", "permisstion_type": "adminProject"},
            {"practice_area": "Development", "permisstion_type": "practiceLeadProject"},
            {"practice_area": "Design", "permisstion_type": "practiceLeadJrProject"}
        ],
        should_attend=[{"practice_area": "Development", "permission_type": "memberProject"}],
        could_attend=[{"practice_area": "Design", "permission_type": "memberGeneral"}],
```

The test assertion will need to be updated for this.

It would allow tests such as this one

```
def test_event_projects_admins_must_attend(event, event_pm):
    projects_admins_must_attend = Event.objects.filter(
        must_attend__contains=[
            {"permission_type": "adminProject"}
        ]
    )

    assert projects_admins_must_attend.count() == 1
    assert len(filter_objects_by_name(projects_admins_must_attend, "All")) == 1
    assert len(filter_objects_by_name(projects_admins_must_attend, "PM")) == 0
```

where I added a fixture for a `PM` meeting that the `adminProject` `should_attend`

```
@pytest.fixture
def event_pm(project):
    return Event.objects.create(
        name="PM",
        project=project,
        must_attend=[
            {"practice_area": "Development", "permission_type": "practiceLeadProject"},
            {"practice_area": "Design", "permission_type": "practiceLeadJrProject"},
        ],
        should_attend=[
            {"practice_area": "Development", "permission_type": "memberProject"},
            {"permission_type": "adminProject", "practice_area": "Design"},
        ],
        could_attend=[{"practice_area": "Design", "permission_type": "memberGeneral"}],
    )
```